src/xrpld/app/ledger/detail/LedgerMaster.cpp

```diff
@@ -467,7 +467,7 @@ LedgerMaster::applyHeldTransactions()
             ApplyFlags flags = tapNONE;
             auto const result =
                 app_.getTxQ().apply(app_, view, it.second, flags, j);
-            if (result.second)
+            if (any | result.applied)
```

Comment:
Sorry, I was not clear enough with  my previous comment. You can use 
`any |= result.applied`
instead of 
```
if (result.second)
   any = true;
```

Reply:
Fixed in 1c0b3a6f8ca9b038035689b68971ad5e31ba30df